tools/topology/topology2/sof-ace-tplg/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Array of "input-file-name;output-file-name;comma separated pre-processor variables"
+set(TPLGS
+# HDMI only topology with passthrough pipelines
+"cavs-passthrough-hdmi\;sof-hda-generic-idisp\;USE_CHAIN_DMA=true"
+# HDA topology with mixer-based pipelines for HDA and passthrough pipelines for HDMI
+"cavs-passthrough-hdmi\;sof-hda-generic\;HDA_CONFIG=mix,USE_CHAIN_DMA=true"
+# If the alsatplg plugins for NHLT are not available, the NHLT blobs will not be added to the
+# topologies below.
+"cavs-passthrough-hdmi\;sof-hda-generic-4ch\;PLATFORM=mtl,\
+HDA_CONFIG=mix,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-generic-4ch.bin"
```

Comment:
missing "cavs-passthrough-hdmi\;sof-hda-generic-2ch\;\
HDA_CONFIG=mix,NUM_DMICS=2,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-generic-2ch.bin,USE_CHAIN_DMA=true"

Reply:
and done